Core.ApplicationServices/UserService.cs

```diff
@@ -344,5 +371,26 @@ private static void Delete(User user)
             user.HasApiAccess = false;
             user.HasStakeHolderAccess = false;
         }
+
+        private Result<UserDeletionStrategy, OperationError> GetUserDeletionStrategy(int orgId, User user)
+        {
+            if (user.Deleted)
```

Comment:
you are missing a check for if authenticated user is Local admin in org identified by orgID OR is global admin.. If not, then  we return forbidden before going any further

Reply:
also rename ti GetOrganizationalUserDeletionStrategy

Reply:
so also rename the enum